app/views/catalog/_search_results.html.erb

```diff
@@ -8,7 +8,7 @@
 <% end %>
 
 <% content_for(:skip_links) do -%>
-    <%= link_to t('blacklight.skip_links.first_result'), '#documents', class: 'd-inline-flex p-2 m-1', data: { turbolinks: 'false' } %>
```

Comment:
shouldn't this be `data: { turbo: false }`?

Reply:
I don't think that turbo needs to skip this. I'm pretty sure it knows that intra-document links don't get cached.